packages/snaps-sdk/src/types/methods/notify.ts

```diff
@@ -0,0 +1,11 @@
+import type { EnumToUnion } from '../../internals';
+
+export enum NotificationType {
+  InApp = 'inApp',
+  Native = 'native',
+}
+
+export type Notify = {
```

Comment:
Is there a reason not to define the params and result in each of these files instead of one large file?

Reply:
Done!